### PR TITLE
⌗104403 Restore Old Cache-Clearing Code

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**⌗104403**](http://central.tri.be/issues/104403)

**Code Review Note:** Please keep in mind that we're trying to keep this change as literally low-cost as possible, and that this is not "new" code I'm writing. We're simply adding preexisting code back in that I originally removed in PR https://github.com/moderntribe/pardot/pull/4/.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -140,6 +140,10 @@ Filter the regular expression used to find URLs to be converted to https://go.pa
 
 == Changelog ==
 
+= 1.4.6 TBD =
```
Comment (barryhughes) [inline README.txt]: Not sure if 1.4.6.1 or 1.4.7 is more appropriate (I'd lean toward the latter for this project ... but it definitely won't be 1.4.6). Might be better to replace with:

`= TBD =`

Comment (barryhughes) [reply README.txt]: (...We can make a decision and fix this before deployment though)

Comment (ggwicz) [reply README.txt]: Totally, this was just a typo on my part (copied and pasted the `=`-formatted header from `1.4.6`, forgot to remove the `1.4.6` string).

File: includes/pardot-settings-class.php [hunk 2]
```diff
@@ -383,7 +383,17 @@ function sanitize_fields( $dirty ) {
 		}
 
 		if ( isset( $_POST['clear'] ) ) {
+
+			global $wpdb;
+
+			$collecttrans = $wpdb->get_col( "SELECT option_name FROM {$wpdb->options} WHERE option_name LIKE '_transient_pardot%';" );
```
[the remark this answers is not in the document]
Comment (ggwicz) [reply includes/pardot-settings-class.php]: Great points!

**Pull request merged**